### ISSUE TITLE
export: enhance bibtex formatter to include conference location for conference papers

### ISSUE THREAD

**Issue by krzysztof** [zenodo/zenodo#855]
At the moment bibtex expor ignores the notion of the conference location.
This, however, is supported by bibtex as "location"/"venue" fields: http://www.pirbot.com/mirrors/ctan/macros/latex/contrib/biblatex/doc/biblatex.pdf

It would be good to parse the information inside ``record["meeting"]["place"]`` and include it inside a "venue" key.

**Comment by krzysztof** [comment]
Additionally to an address, an explicit field stating that given record is a conference presentation might be userful, e.g. a 'howpublished/note' field (with value "conference"?).

**Comment by dosydon** [comment]
Hi. I'm interested in working on this issue. Adding "venue" keys seems easy. I believe we just need to add "_get_place(self)" function to Bibtex class (from bibtex.py) and edit other related parts like "_fetch_fields" function. 

As for adding descriptions to "howpublish/note" field, having "@inproceedings" entries with something like "Proceedings of nth Conference on ..." as "booktitle"  sounds much better.  Or are you talking about cases where a given conference paper record lacks "booktitle" field (in which case, the current implementation seems to fall back to "@proceedings" type)?

**Comment by krzysztof** [comment]
@dosydon Yes, the "venue" is straightforward to add (although it should be ``_get_venue``).

The howpublish/note field was meant to distinguish between conference papers and conference posters (at the moment there is no clear way on how to do it in bibtex, even though our deposits can be of type "Poster").

The idea was to include "howpublished" with a value such as "Poster presentation at {location} on {date}", similarly to how it's suggested here: http://tex.stackexchange.com/a/184581

**Comment by dosydon** [comment]
@krzysztof  For a starter, I made a PR that adds "venue" field to @inproceeding (https://github.com/zenodo/zenodo/pull/1031). 
I'll work on the second issue after I get used to the workflow with the first PR.
